src/marketing/shopping-tools.md

```diff
@@ -1,8 +1,40 @@
 ---
 title: Shopping Tools
+redirect_from:
+  - /marketing/customer-engagement.html
+sections:
+ - title: Email a Friend
+   content: The Email a Friend link makes it easy for your customers to share links to products with their friends.
+   url: /marketing/email-a-friend.html
+
+- title: Wish List
+   content: A wish list is a list of products that a registered customer can share with friends, or save to transfer to the cart at a later date
+   url: /marketing/email-a-friend.html
+
+- title: Compare Products
+   content: The Compare Products block lets your customers quickly compare the features of one product with another.
+   url: /marketing/product-compare.html
+
+ - title: Product Reviews
+   content: Product reviews help build a sense of community, and are considered to be more credible than any advertising money can buy.
+   url: /marketing/product-reviews.html
+
+ - title: dotdigital Engagement Cloud
+   content: Use dotdigital Engagement Cloud to automate professional, personalized communications and generate reports using data from your Magento store.
+   url: /marketing/dotdigital/engagement-cloud.html
+
+ - title: Image zoom
+   content: When zoom is activated, customers can click the main image and move the cursor around to magnify different parts of the image.
+   url: /catalog/media-browser.html
+
+ - title: Stock Availability
+   content: Helps to notify the buyer about the availability of a product. Possibly a create the **Notify me when this product is in the stock** link for every product that is out of stock.
```

Comment:
Markdown formatting does not work for the "tiles". COn use HTML formatting, or just simplify for aa smalll blub of text.
```suggestion
   content: It helps buyers when you notify them about the availability of a product, and can result in increased sales. You can create the "Notify me when this product is in stock" link for every product that is out of stock.
```

Reply:
Thank you for your notice, @leslietilling ☀️
I have fixed this, please approve 😊